tutorials/Homology-Modelling/homology-modelling-tutorial.ipynb

```diff
@@ -597,7 +611,7 @@
     "\n",
     "class MyModel(automodel):\n",
     "    def special_patches(self, aln):\n",
-    "        # A disulfide between residues 196 and 197:\n",
+    "        # A disulfide between residues 196 and 197: - we only fix one here...\n",
```

Comment:
To avoid reader confusion, I'd suggest changing this slightly (or at least removing the elipses)
```suggestion
    "        # A disulfide between residues 196 and 197: - for brevity we only fix one here.\n",
```

Reply:
done